news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,240 @@
+
+#### Back-Office improvements
+
+- TinyMCE mobile theme is enabled in the Back-Office [#23225](https://github.com/PrestaShop/PrestaShop/issues/23225)
+- The overrides done in the shop are displayed in the Informations tab [#21368](https://github.com/PrestaShop/PrestaShop/issues/21368)
+- A new order internal note is available in the Order View Page [#14753](https://github.com/PrestaShop/PrestaShop/issues/14753)
+- Back Office users can upload custom avatars [#18653](https://github.com/PrestaShop/PrestaShop/issues/18653)
+
+### Notable fixes
+
+### **Topwatchers:**
+
+- When a product had a specific price applied and was discounted by unit, the discount displayed on the product page was false [#16163](https://github.com/PrestaShop/PrestaShop/issues/16163) (Major)
+- When Clear cache button was clicked twice, the user would see an error page [#11105](https://github.com/PrestaShop/PrestaShop/issues/11105) (Major)
+- Profiler was not compatible with hooks and module [#9659](https://github.com/PrestaShop/PrestaShop/issues/9659)
+- Double click on "Add to cart" on the product page FO was freezing the page [#9634](https://github.com/PrestaShop/PrestaShop/issues/9634) (Major)
+
+### Major bugs:
+
+**Front-Office**
+
+- Guess Order Tracking redirects to shop Homepage when the Friendly URLs are disabled [#20194](https://github.com/PrestaShop/PrestaShop/issues/20194)
+- An error is displayed on the checkout if the Legal compliance module is installed [#12509](https://github.com/PrestaShop/PrestaShop/issues/12509)
+- Checkout slowdown when many orders are made with a Guest account [#16584](https://github.com/PrestaShop/PrestaShop/issues/16584)
+- Ecotax is displayed tax excl instead of tax incl in FO [#18835](https://github.com/PrestaShop/PrestaShop/issues/18835)
+- Confirmation button on Checkout should be disabled if Terms and conditions are not approved [#19161](https://github.com/PrestaShop/PrestaShop/issues/19161)
+
+**Back-Office**
+
+- An exception is thrown when adding an invalid parameter to the Language form and "No picture" image [#22500](https://github.com/PrestaShop/PrestaShop/issues/22500)
+- In Back-Office, some of the js parts rely on form field's ids and attributes that are modified since 1.7.8 [#21819](https://github.com/PrestaShop/PrestaShop/issues/21819)
+- The number of characters is only checked by the Front-End [#19505](https://github.com/PrestaShop/PrestaShop/issues/19505)
+- Possibility to create catalog price rule with a percentage value greater than 100% [#19013](https://github.com/PrestaShop/PrestaShop/issues/19013)
+- JS issue when editing an image in TinyMCE [#11011](https://github.com/PrestaShop/PrestaShop/issues/11011)
+- Unit price reset to 0 if you activate/deactivate or duplicate products from the Product List [#10792](https://github.com/PrestaShop/PrestaShop/issues/10792)
+- Issues when installing PrestaShop with SSL [#10482](https://github.com/PrestaShop/PrestaShop/issues/10482)
```

Comment:
```suggestion
- Installing PrestaShop with SSL was complex because some settings were not acknowledged [#10482](https://github.com/PrestaShop/PrestaShop/issues/10482)
```